# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/CredentialManagement.webidl
# Commit: ccd96b99aea2
# Full Hash: ccd96b99aea28e8214e585b864d3fb2245bac142
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-06-07 08:31:23
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing failures complaining about fedcm-no-registered-idps.https.html.
#   
#   This reverts commit ba33457e0cbac24a8b7d963c45138c62e92a74ae.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4bfadfc33207 -r ccd96b99aea2 dom/webidl/CredentialManagement.webidl
--- a/dom/webidl/CredentialManagement.webidl	Fri Jun 06 21:07:41 2025 +0300
+++ b/dom/webidl/CredentialManagement.webidl	Fri Jun 06 21:35:18 2025 +0300
@@ -54,5 +54,7 @@
   // https://w3c.github.io/webauthn/#sctn-credentialcreationoptions-extension
   [Pref="security.webauth.webauthn"]
   PublicKeyCredentialCreationOptions publicKey;
+  [Pref="dom.security.credentialmanagement.identity.enabled"]
+  IdentityCredentialInit identity;
   AbortSignal signal;
 };